apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -53,5 +74,46 @@ func validateRAID(r *RAIDConfig) error {
 		return fmt.Errorf("hardwareRAIDVolumes and softwareRAIDVolumes can not be set at the same time")
 	}
 
+	if bmcAccess == nil || bmcAccess.RAIDInterface() == "no-raid" {
```

Comment:
According to the discussion in here https://github.com/metal3-io/baremetal-operator/pull/1040/files#r755873502, if bmc address is nil(thus bmcAccess is nil), we will allow pass through.